SConstruct

```diff
@@ -197,9 +203,10 @@ def scons():  # pylint: disable=too-many-locals,too-many-branches
             version = tag
 
         try:
+            # pylint: disable-next=consider-using-with
```

Comment:
I realize you're not trying to fix all pylint warnings in this PR, but if we have to modify a line to disable the warning, I wonder if we should just modify a line to fix the warning. (at least for simple ones)

Reply:
In most cases I did, it's the lack of testing for this code-block that stopped me in this case.